src/ck-core/cklocation.C

```diff
@@ -2122,6 +2122,9 @@ CkLocMgr::~CkLocMgr() {
 }
 
 void CkLocMgr::pup(PUP::er &p){
+  if (p.isPacking() && pendingImmigrate.size() > 0)
+    CkAbort("Attempting to pup location manager with buffered migration messages."
+            " Likely cause is checkpointing before array creation has fully completed\n");
 	IrrGroup::pup(p);
```

Comment:
Can you fix the indentation of the lines below the 2 lines you added? Otherwise, indentation might look weird depending on the tab size that the editor uses.

Reply:
I agree, but I really think a separate patch should clean up the indentation of that file, or at least the pup method.

Reply:
A short-term option could be to just keep the tab-based indentation for the added lines.

Reply:
I considered that, but figured that the CkAbort indentation would look inconsistent with tabs

Reply:
FWIW, I already have a commit that reformats much of CkLocMgr, including pup. It's part of a larger overhaul of CkLocMgr that I'm not pushing anywhere until after 6.10 is done and I've done a bit more cleanup on it.